### PR TITLE
init 1.32 release

### PR DESCRIPTION
**Pull request by bschimke95**
Initialize the 1.32 release branch with the current release candidate.

### CODE REVIEW

File: .github/workflows/integration.yaml [hunk 1]
```diff
@@ -107,7 +107,7 @@ jobs:
           TEST_VERSION_UPGRADE_MIN_RELEASE: "1.31"
           TEST_MIRROR_LIST: '[{"name": "ghcr.io", "port": 5000, "remote": "https://ghcr.io", "username": "${{ github.actor }}", "password": "${{ secrets.GITHUB_TOKEN }}"}, {"name": "docker.io", "port": 5001, "remote": "https://registry-1.docker.io", "username": "", "password": ""}, {"name": "rocks.canonical.com", "port": 5002, "remote": "https://rocks.canonical.com/cdk"}]'
         run: |
-          cd tests/integration && sg lxd -c 'tox -e integration -- --tags pull_request'
+          cd tests/integration && sg lxd -c 'tox -e integration --'
```
Comment (bschimke95) [inline .github/workflows/integration.yaml:110]: Run the full test suite to ensure stuff does not break. (We have a card to automatically do this for release branches in the future).

Comment (bschimke95) [reply .github/workflows/integration.yaml:110]: Remove once the tests pass.

Comment (bschimke95) [reply .github/workflows/integration.yaml:110]: resolved

**Review by HomayoonAlimohammadi — APPROVED** [review]
Thanks a lot @bschimke95, LGTM!

**Review by louiseschmidtgen — APPROVED** [review]
LGTM

**Pull request merged**